### PR TITLE
Add USA Contacts scraper, some merged data

### PR DESCRIPTION
**Pull request by geramirez**
The records below are affecting the ids and acronyms. issue146 can't be merged till these are resolved.  
-National Oceanic and Atmospheric Administration
-National Park Service
-Bureau of Indian Affairs
-Department of Housing and Urban Development
-U.S. Citizenship & Immigration Services

### CODE REVIEW

**Comment by rjmajma** [comment]
Is the issue around the acronyms for the listed agencies?

It should be: 
-NOAA
-NPS
-[none]
-HUD
-USCIS & CIS

File: contacts/data/GSA.yaml [hunk 1]
```diff
@@ -20,10 +24,6 @@ departments:
 description: GSA's mission is to use expertise to provide innovative solutions for
   our customers in support of their missions and by so doing, foster an effective,
   sustainable, and transparent government for the American people.
-common_requests:
```
Comment (konklone) [inline contacts/data/GSA.yaml:23]: @khandelwal This `common_requests` array should get moved to the manual data, right?

File: contacts/data/OSTP.yaml [hunk 2]
```diff
@@ -18,12 +20,11 @@ description: The mission of the Office of Science and Technology Policy is three
   ensure that the policies of the Executive Branch are informed by sound science;
   and third, to ensure that the scientific and technical work of the Executive Branch
   is properly coordinated so as to provide the greatest benefit to society.
-common_requests:
```
Comment (konklone) [inline contacts/data/OSTP.yaml:21]: Ditto to this OSTP data.

**Comment by geramirez** [comment]
It's a couple of issues [issue 167](https://github.com/18F/foia-hub/issues/167)

**Comment by konklone** [comment]
This looks great. I'm unable to reproduce this locally -- when I run this, I lose the `top_level` fields that were added by @khandelwal's manual data. I want to move this forward, so am merging it, but I feel like we still have some entropy to our data loading process that we can't quite seem to squash.

File: contacts/data/GSA.yaml (hunk 1 above)
Comment (khandelwal) [inline contacts/data/GSA.yaml:23]: It _is_ in the manual data: https://github.com/18F/foia/blob/master/contacts/manual_data/GSA.yaml#L1

File: contacts/data/OSTP.yaml (hunk 2 above)
Comment (khandelwal) [inline contacts/data/OSTP.yaml:21]: Same with the OSTP related stuff: https://github.com/18F/foia/blob/master/contacts/manual_data/OSTP.yaml#L1

Comment (khandelwal) [inline contacts/data/OSTP.yaml:21]: When I run scraper.py common_requests shows up in the YAML file. It's probably getting taken out at a different stage.

**Pull request merged**